File: Modul 3/Soal 4/PRAK304-2410817210019-AchmadReihanAlfaiz.c
#include <stdio.h>

int main()
{
     int userInput;
     scanf("%d", &userInput);

     printf("\nOutput: \n");

     if (userInput == 10 || (userInput >= 20 && userInput < 100))
     {
          printf("Puluhan");
     }
     else if (userInput >= 11 && userInput <= 19)
     {
          printf("Belasan");
     }
     else if (userInput >= 1 && userInput <= 9)
     {
          printf("Satuan");
     }
     else if (userInput == 0)
     {
          printf("Nol");
     }
     else
     {
          printf("Anda Menginput Melebihi Limit Bilangan");
     }

     return 0;
}

File: Modul 3/Soal 4/PRAK304-2410817210019-AchmadReihanAlfaiz.py
user_input = int(input())

print("\nOutput: ")

if user_input == 10 or (user_input >= 20 and user_input < 100):
    print("Puluhan")
elif user_input >= 11 and user_input <= 19:
    print("Belasan")
elif user_input >= 1 and user_input <= 9:
    print("Satuan")
elif user_input == 0:
    print("Nol")
else:
    print("Anda Menginput Melebihi Limit Bilangan")
